### DISCUSSION TITLE
How to set accent color, widgets and  size using CSS ?

### DISCUSSION THREAD

**Discussion by KaKi87** [jarek-foksa/xel#117] [Q&A] [answered]
*EDIT: sorry, I forgot you enabled the discussions section, it's still so rare nowadays. Feel free to move this there.*

Thanks

**Comment by jarek-foksa** [comment, marked as answer]
In order to change the accent color of a specific element or subtree you will have to override custom properties such as ` --accent-color`, `--light-accent-color` and `--dark-accent-color`.

To change the size via CSS you should just override the `width`, `height`, `min-width`, `min-height` or similar properties.

Generally I do not recommend doing any of this as it will make your UI look inconsistent.

**Reply by KaKi87** [reply]
Well, I'm looking for CSS-based solutions because Vue bundlers don't support the `meta` tags solution.

**Reply by jarek-foksa** [reply]
I see, Vue does not support Xel then.
